npm-downloads-ts/DEVELOP.md

```diff
@@ -0,0 +1,41 @@
+# How to use
+
+First, you should read [Develop Apps Script using TypeScript]. That guide
+explains how [`clasp`] can be used to develop an Apps Script project using
+TypeScript.
+
+## Workflow
+
+1.  Clone this project:
+
+    ```sh
+    git clone https://github.com/googledatastudio/example-connectors.git
+    ```
+
+1.  Change to the `npm-downloads-ts` directory:
+
+    ```sh
+    cd npm-downloads-ts
+    ```
+
+1.  Update the `scriptId` property in `.clasp.json` to an Apps Script project
```

Comment:
Nit: might be worth having more detail here, like where to find the exact script ID